service/controller/resource/app/desired.go

```diff
@@ -152,6 +151,11 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) ([]*g8s
 		return nil, microerror.Mask(err)
 	}
 
+	appOperatorVersion, err := r.getComponentVersion(fmt.Sprintf("v%s", clusterConfig.ReleaseVersion), "app-operator")
```

Comment:
We keep releases version in clusterconfig w/o prefix `v`.

Reply:
Why do we need the `v` prefix? We don't use that in project.go.

Reply:
In `AzureClusterConfigs`, it doesn't contain `v` in release version, 
```
k get azureclusterconfigs t2x8x-azure-cluster-config -o yaml
..
  versionBundle:
    version: 0.23.15-dev
```
But all `release` CR starts with `v` as you can see below. 
```
k get releases
NAME           KUBERNETES VERSION   STATE        AGE
v108.108.108   1.17.9               active       19d
v11.3.0        1.16.8               deprecated   104d
v11.3.1        1.16.8               deprecated   88d
v11.3.3        1.16.8               deprecated   59d
v11.4.0        1.16.12              active       47d
v12.0.0        1.17.9               active       19d
v123.0.0       1.16.10              wip          67d
v42.0.0        1.17.9               wip          10d
```

Reply:
Ah OK because it's in the releases CR name. I get this now but what about adding a comment to explain?